### DISCUSSION TITLE
Introducing Myself

### DISCUSSION THREAD

**Discussion by VenkataSatyaSrikarParvatha** [community/community#110825] [GitHub Education]
### Hi everyone,

Hi everyone, my name is VENKATA SATYA SRIKAR PARVATHA and at present I am pursuing my Bachelor of Technology in KL (DEEMED TO BE UNIVERSITY). It's happy to be part of this community.

**Comment by gubernari** [comment]
Welcome

**Comment by SebastienXia** [comment]
Welcome to Github community dude, and if you want learn more about Github, do not hesitate to consult the Github documentation:

[]()https://docs.github.com/en

enjoy ! 👋

**Comment by drdixit** [comment]
Just dropped by to say 👋Hello! @VenkataSatyaSrikarParvatha
